scripts/lib/python/plugins/analytics.py

```diff
@@ -25,7 +25,7 @@
 	 viewBox="0 0 505 505" style="enable-background:new 0 0 505 505;" xml:space="preserve">
```

Comment:
This file have changed.. You need to rebase your branch off my branch

Reply:
Yes, I need to some work on this PR before it merges. I should be able to get to it this week.